spec/ruby-mpd/parser_spec.rb

```diff
@@ -89,9 +89,9 @@
     end
 
     context "with :db_update key" do
-      expected_time = Time.local(2015, 6, 11, 21, 14, 33)
-      it { expect(subject.send(:parse_key, :db_update, '1434024873'))
-        .to eql(expected_time) }
+      expected_time = Time.at(1434024873)
+      it { expect(subject.send(:parse_key, :db_update, '1434024873').to_i)
```

Comment:
It looks like ```Time.at(i)``` in this instance returns a local time represented like ```2015-06-11 13:14:33 +0100```. The documentation states:

> If a numeric argument is given, the result is in local time

So I think you need to convert the object back to UTC or in my case Epoch, to guarantee they will be the same.